transifex/push.py

```diff
@@ -46,6 +46,10 @@ def parse_arguments():
         help='URL to use to clone the repository.'
     )
     parser.add_argument(
+        'repo_owner',
+        help='git team responsible for translation jobs.'
```

Comment:
Nit: Maybe include that this is the user/team that will be pinged when errors occur?